### ISSUE TITLE
train.grpo.py里虽然import了OneSearchReward, 但是根本没用。。。

### ISSUE THREAD

**Issue by SmurfetteY** [benchen4395/onesearch-family#4]
from reward import OneSearchReward

# Reward function
def composite_item_reward_func(
    prompts, completions, completion_ids=None, **kwargs,
):
    """R_item = R_Rel + R_CTR + R_C&O  (OneSearch-V2 §4.5).

    TRL passes per-sample fields of length B*G — each prompt's per-prompt
    fields (`relevance`, `ctr`, `order_sids`, `click_sids`) are duplicated G
    times. We therefore index by the global sample index `i`.
    """
    relevance = kwargs.get("relevance")
    ctr = kwargs.get("ctr")
    order_sids = kwargs.get("order_sids")
    click_sids = kwargs.get("click_sids")

    rewards = []
    for i, c in enumerate(completions):
        rel = float(relevance[i].get(c, 0.0)) if relevance is not None else 0.0
        cv  = float(np.clip(ctr[i].get(c, 0.0), 0.0, 1.0)) if ctr is not None else 0.0
        if order_sids is not None and c in order_sids[i]:
            co = 4.0
        elif click_sids is not None and c in click_sids[i]:
            co = 3.0
        else:
            co = 0.0
        rewards.append(rel + cv + co)
    return rewards
......
# Main
def main():
......
    reward_funcs = [composite_item_reward_func]

    trainer = OneSearchGRPOTrainer(
        model=model,
        processing_class=tokenizer,
        args=training_args,
        reward_funcs=reward_funcs,
......

    reward_funcs = [composite_item_reward_func]
...
